packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js

```diff
@@ -498,18 +498,37 @@ testModule('API', moduleConfig, () => {
     });
 
     testModule('converter option', () => {
-        test('toHtml and fromHtml should be called once after value option changed', function(assert) {
+        ['v', ''].forEach(initValue => {
+            test(`toHtml and fromHtml must be called the correct number of times on init if value is '${initValue}'`, function(assert) {
+                const converter = {
+                    toHtml: sinon.stub().callsFake((e) => e),
+                    fromHtml: sinon.stub().callsFake((e) => e),
+                };
+
+                this.options = {
+                    converter,
+                    value: initValue,
+                };
+
+                this.createEditor();
+
+                assert.strictEqual(this.options.converter.toHtml.callCount, 1);
+                assert.strictEqual(this.options.converter.fromHtml.callCount, 0);
+            });
+        });
+
+        test('toHtml and fromHtml must be called the correct number of times after value option changed', function(assert) {
             const converter = {
-                toHtml: sinon.stub(),
-                fromHtml: sinon.stub(),
+                toHtml: sinon.stub().callsFake((e) => e),
+                fromHtml: sinon.stub().callsFake((e) => e),
             };
             this.options = { converter };
 
             this.createEditor();
 
             this.instance.option('value', 'new value');
 
-            assert.strictEqual(this.options.converter.toHtml.callCount, 1);
+            assert.strictEqual(this.options.converter.toHtml.callCount, 2);
```

Comment:
looks strange

Reply:
Hi, I tried using the onInitialised event, but it didn't work properly: the call history was not reset. Now I wrote this immediately right after createEditor: [494ad1eb4f2859846b86d863f2cd5b748e62b209]